include/PR/os_voice.h

```diff
@@ -13,18 +13,18 @@ typedef enum OsVoiceHandleMode {
 } OsVoiceHandleMode;
 
 typedef struct {
-    /* 0x0 */ OSMesgQueue* mq;
-    /* 0x4 */ s32 channel; // Controller port
-    /* 0x8 */ OsVoiceHandleMode mode;
-    /* 0xC */ u8 status;
+    /* 0x0 */ OSMesgQueue* __mq;
+    /* 0x4 */ s32 __channel; // Controller port
+    /* 0x8 */ s32 __mode;
+    /* 0xC */ u8 cmd_status;
 } OSVoiceHandle; // size = 0x10
 
 typedef struct {
     /* 0x00 */ u16 warning;       // Warning
-    /* 0x02 */ u16 answerNum;     // Candidate number (0~5)
-    /* 0x04 */ u16 voiceLevel;    // Voice input level
-    /* 0x06 */ u16 voiceRelLevel; // Relative voice level "voice_sn"
-    /* 0x08 */ u16 voiceTime;     // Voice input time
+    /* 0x02 */ u16 answer_num;     // Candidate number (0~5)
+    /* 0x04 */ u16 voice_level;    // Voice input level
+    /* 0x06 */ u16 voice_sn; // Relative voice level
+    /* 0x08 */ u16 voice_time;     // Voice input time
```

Comment:
Tiny nit: can you make these aligned again?
```suggestion
    /* 0x04 */ u16 voice_level;    // Voice input level
    /* 0x06 */ u16 voice_sn;       // Relative voice level
    /* 0x08 */ u16 voice_time;     // Voice input time
```